lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -2195,6 +2259,9 @@ def set_alert_rule_data(self, name: str, unit_rules: dict, label_rules: bool = T
         The unit rules should be a dict, which is has additional Juju topology labels added. For
```

Comment:
```suggestion
        The unit rules should be a dict, which has additional Juju topology labels added. For
```

Drive-by change. Not really part of the PR scope. Do with it what you want. :)